swath_projector/earthdata_varinfo_config.json

```diff
@@ -48,16 +71,16 @@
     {
       "Applicability": {
         "Mission": "TEMPO",
-        "ShortNamePath": "TEMPO_O3TOT_L2",
-        "VariablePattern": "^/product/.*|^/support_data/.*|^/geolocation/(?!time$).*"
+        "ShortNamePath": "TEMPO_.*_L2",
+        "VariablePattern": "^/(product|support_data|geolocation|qa_statistics)/(?!time$).*"
```

Comment:
`qa_statistics` is a group that appears to be in V3 of the TEMPO products, but not older versions.